internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/PartialUpdateTest.java

```diff
@@ -140,6 +142,139 @@ public void setUp() {
     this.parentController = parentControllers.get(0);
   }
 
+  @Test
+  public void testReproduceError() throws IOException {
+    final String storeName = Utils.getUniqueString("reproduce");
+    String parentControllerUrl = parentController.getControllerUrl();
+    Schema keySchema = AvroCompatibilityHelper.parse(loadFileAsString("UserKey.avsc"));
+    Schema valueSchema = AvroCompatibilityHelper.parse(loadFileAsString("UserValue.avsc"));
+    Schema writeComputeSchema = WriteComputeSchemaConverter.getInstance().convertFromValueRecordSchema(valueSchema);
+
+    try (ControllerClient parentControllerClient = new ControllerClient(CLUSTER_NAME, parentControllerUrl)) {
+      assertCommand(
+          parentControllerClient.createNewStore(storeName, "test_owner", keySchema.toString(), valueSchema.toString()));
+      UpdateStoreQueryParams updateStoreParams =
+          new UpdateStoreQueryParams().setStorageQuotaInByte(Store.UNLIMITED_STORAGE_QUOTA)
+              // .setPartitionCount(4)
```

Comment:
Can we remove all the commented lines?

Reply:
Yeah thanks for the review, I need to cleanup those debug logs/commnets and random names before this is ready.